regression/deterministic_shuffling.yaml

```diff
@@ -7,22 +7,22 @@ command: |-
   cd streaming
   pip install -e '.[dev]'
 
-  python regression/synthetic_dataset.py --create --out /tmp/streaming_dataset/
+  python regression/synthetic_dataset.py --create --out /tmp/streaming_dataset/ --num-samples 10000
```

Comment:
This still generates one shard file, can you add `--size_limit 4096` so that it generates more than 50 shard files ?